ci/test/pipeline.template.yml

```diff
@@ -78,7 +78,7 @@ steps:
         depends_on: []
         timeout_in_minutes: 60
         agents:
-          queue: builder-linux-aarch64
+          queue: builder-linux-aarch64-mem
```

Comment:
I assumed we'd switch the builder-linux-aarch64 over to r8g if it works well. If we want to use the mem builder here, have to adapt it in other pipelines too.

Reply:
Makes sense! Switched over all of the existing pipelines to the `-mem` builders